### PR TITLE
Numerical function docstring

### PR DESCRIPTION
**Pull request by deepaksidhu**

### CODE REVIEW

File: R/numerical_eda.R [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+#'
+#' This function takes in a tibble or dataframe object, two numeric columns,
+#' and produces either a scatter or line plot to visualize the relationship
+#' between the two numerical features. Users can optionally change default
+#' arguments for plot-type, color, title, size of text, color-scheme, and
+#' toggle log transformation for the x and y axis.
+#'
+#' @param data A tibble or data frame object.
+#' @param xval A character variable used to represent the x-axis.
+#' @param yval A  variable used to represent the y-axis.
```
Comment (nichowu) [inline R/numerical_eda.R]: character variable

File: R/numerical_eda.R [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+#' @param plot_type An optional character variable used to represent
+#' the graphical relationship between xval and yval,
+#' options are scatter or line plot.
```
Comment (nichowu) [inline R/numerical_eda.R]: options are "scatter" or "line".

File: R/numerical_eda.R [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+#' @param color An optional character variable used to group the data
+#' points in different colors.
+#' @param title An optional character variable used to set the title of the plot.
+#' @param font_size An optional integer variable used to set the font size.
```
Comment (nichowu) [inline R/numerical_eda.R:18]: specify font size for the plot title and axis.

**Review by nichowu — CHANGES_REQUESTED** [review]
Changes requested

File: R/numerical_eda.R (hunk 1 above)
Comment (deepaksidhu) [reply R/numerical_eda.R]: Updated

File: R/numerical_eda.R (hunk 3 above)
Comment (deepaksidhu) [reply R/numerical_eda.R:18]: I have updated it

**Review by nichowu — APPROVED** [review]

**Pull request merged**